src/crimson/osd/pg.h

```diff
@@ -477,6 +477,10 @@ class PG : public boost::intrusive_ref_counter<
     void trim(const pg_log_entry_t &entry) override {
       // TODO
     }
+    void partial_write(pg_info_t *info, const pg_log_entry_t &entry) override {
```

Comment:
The crimson crash is coming from this line. See https://github.com/ceph/ceph/pull/62522/files#r2050803678.